.github/workflows/cmd.yml

```diff
@@ -0,0 +1,417 @@
+name: Command
+
+on:
+  issue_comment: # listen for comments on issues
+    types: [ created ]
+
+permissions: # allow the action to comment on the PR
+  contents: write
+  issues: write
+  pull-requests: write
+  actions: read
+
+jobs:
+  is-org-member:
+    if: startsWith(github.event.comment.body, '/cmd')
+    runs-on: ubuntu-latest
+    outputs:
+      member: ${{ steps.is-member.outputs.result }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Generate token
+        id: generate_token
+        uses: tibdex/github-app-token@v2.1.0
+        with:
+          app_id: ${{ secrets.CMD_BOT_APP_ID }}
+          private_key: ${{ secrets.CMD_BOT_APP_KEY }}
+
+      - name: Check if user is a member of the organization
+        id: is-member
+        uses: actions/github-script@v7
+        with:
+          github-token: ${{ steps.generate_token.outputs.token }}
+          result-encoding: string
+          script: |
+            const fs = require("fs");
+            try {              
+              const org = '${{ github.event.repository.owner.login }}';
+              const username = '${{ github.event.comment.user.login }}';
+            
+              const membership = await github.rest.orgs.checkMembershipForUser({
+                  org: org,
+                  username: username
+              });
+            
+              console.log(membership, membership.status, membership.status === 204);
+            
+              if (membership.status === 204) {
+                return 'true';
+              } else {
+                console.log(membership);
+                fs.appendFileSync(process.env["GITHUB_STEP_SUMMARY"], `${membership.data && membership.data.message || 'Unknown error happened, please check logs'}`);
```

Comment:
What is this doing?

Reply:
it sets the GITHUB_STEP_SUMMARY env to see the reason of fail in the job summary
https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/workflow-commands-for-github-actions#adding-a-job-summary

![image](https://github.com/user-attachments/assets/3440f617-7eb6-4604-a5d7-1c6ca288d33b)